build-tools-internal/src/main/groovy/elasticsearch.bwc-test.gradle

```diff
@@ -33,7 +33,7 @@ tasks.register("bwcTest") {
 
 plugins.withType(ElasticsearchTestBasePlugin) {
   tasks.withType(Test).configureEach {
-    onlyIf { project.bwc_tests_enabled }
+    onlyIf { project.bwc_tests_enabled || name.contains('#bwcTest') == false }
```

Comment:
This change was suggested by Mark Vieira in order to be able to run both `bwc` and `javaRestTest`s in the same project.